data/datasets/corruptedData/sectionAnomaly/README.md

```diff
@@ -0,0 +1,8 @@
+# Corrupted data - section anomaly
+
```

Comment:
add better explanation. 
"Unlike point-anomaly, in this category we focus on continuous intervals where there is a lot of anomalies, then is not, then is again etc."